platform-coop-toolkit/coop-library-framework
# Make default rights 'Not specified' (resolves #139)

## Description

Changes default rights to "Not specified".

## Steps to test

1. Create new resource.
2. Save.

**Expected behavior:** Default rights should be set to "Not specified".

## Additional information

Not applicable.

## Related issues

- Resolves #139.
